playgrounds/jquery/index.html

```diff
@@ -8,28 +8,99 @@
 <body>
     <div id="examples">
         <script src="../../packages/jquery/lib/bundle.debug.js" charset="utf-8"></script>
+
         <div id="cmp">
             <script>
                 (function() {
                     new window.DevExpress.ui.dxRadioButton(document.getElementById('cmp'), { defaultChecked: false, value: 'radio-example', label: 'Check me' })
                 })()
             </script>
         </div>
+
```

Comment:
Suggestion: Split index.html to radio-button.html and radio-group.html

Reply:
Agree, but let's do it in a separate PR